catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java

```diff
@@ -189,7 +189,8 @@ private Object executeRemoveAllFromStore() throws Exception {
             console.flush();
 
             // Break out if there are no more records to delete
-            if (amountDeleted < batchSize || batchSize < 1) {
```

Comment:
I agree with @emanns95. Not every use of a number in code is a magic red flag. Sometimes they really do just represent obvious, logical assumptions.